sec/review.tex

```diff
@@ -1081,8 +1078,19 @@ \subsubsection{Healthcare settings}
 downfall of an initiative, such as with the NHS England \emph{care.data}
 project~\cite{Carter2015}.
 
-% TODO discussion of:
-%   - trust in models
-%   - calls for best practice and unifying guidelines
-%   - privacy (data synthesis),
-%   - human intervention and feedback
+A workaround to the potential misuse of patient data is to synthesise new
+datasets from existing, confidential data. The methods by which data synthesis
+is achieved are various, but include bespoke, often statistical
+models~\cite{Dahmen2019,Dube2014,McLachlan2016,Tucker2020} and, recently,
+generative adversarial networks (GANs)~\cite{Avino2018,Torfi2020}. Using
+synthetic datasets provides another level of anonymity to the patients, but
+their efficacy against real data has been contested. In a recent
```

Comment:
I'm not sure what `efficacy against real data` means here. Would something like: 

```suggestion
their but their validity as representations of real data has been contested. In a recent
```

work?

Reply:
That's much clearer, yes.